src/public/modules/forms/admin/controllers/edit-start-page-modal.client.controller.js

```diff
@@ -43,8 +43,14 @@ function EditStartPageController(
 
   vm.saveStartPage = function (isValid) {
     vm.hasClickedSave = true
+    const logoState = vm.myform.startPage.logo.state
 
     if (isValid) {
+      if (logoState !== FormLogoState.Custom) {
+        vm.myform.startPage.logo = {
+          state: logoState,
+        }
+      }
```

Comment:
this should be done after the update is successful, right?

Reply:
no, this should be done _prior_. This is because there's additional properties sent on the request, which prevents stricter joi validation.